core/domain/html_validation_service.py

```diff
@@ -135,6 +135,312 @@ def wrap_with_siblings(tag, p):
 BLOCK_COMPONENT_TAG_NAMES = (
     rte_component_registry.Registry.get_block_component_tag_names())
 
+# See https://perso.crans.org/besson/_static/python/lib/python2.7/encodings/cp1252.py # pylint: disable=line-too-long
+# Useful reading: https://www.regular-expressions.info/unicode8bit.html
+CHAR_MAPPINGS = [
```

Comment:
Is this issue only available with states? Do we need to migrate other entity which contains thymol? (Example suggestion, topics etc.)

Reply:
Ouestion suggestions, skills will this problem and I've added a migration for those.